README.md

```diff
@@ -8,4 +8,17 @@
   
   2.And also  setup the [virtualenv](https://docs.python.org/3/tutorial/venv.html) in your local directory.
 
-  3.And finally run the application with command as "python3 -m flask run"
\ No newline at end of file
+  3.And finally run the application with command as "python3 -m flask run"
+
+  ## How to Run:
+   1.Once a collaborator pulls down your project - they can then install a fresh python environment that is,
+
+      "python3 -m venv local_python_environment"
+    
+    2.And then activate that environment and install from your requirements.txt which you have included in your version control:
+
+       "source local_python_environment/bin/activate"
```

Comment:
So here it will be just `source venv/bin/activate` instead of long command

Reply:
yes , that's correct